ai_tools/elasticgraph-mcp-server/src/elasticgraph_mcp/helpers.py

```diff
@@ -17,7 +17,6 @@
 from elasticgraph_mcp.errors import create_command_error, create_not_in_project_error
 
 # Constants
-MIN_RUBY_VERSION = "3.3.0"
```

Comment:
I was going to change this to `3.4.0` but then I noticed that it's unused so it seemed vestigial and safe to remove.